### PR TITLE
Fix: Added bounding box to map, fixed issue with boundary and OSM 403 network errors.

### PR DESCRIPTION
**Pull request by James-Nitron**
Bounding box, signed off by Lynsey.
The map no longer allows you to pan across the world and is locked to England and Wales borders. This fixes boundary and OSM network errors from flooding the console.

### CODE REVIEW

**Comment by James-Nitron** [comment]
Before approving need sign off from Lynsey and wait for the master to be updated with another PR.

File: src/ui/map/TileSet.svelte [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <script>
   import { getContext, setContext } from "svelte";
+  import config from "../../config";
```
Comment (vmilitaru) [inline src/ui/map/TileSet.svelte:3]: If config is not used , probably we can remove the import.

Comment (James-Nitron) [reply src/ui/map/TileSet.svelte:3]: My bad missed that, yep will remove it.

File: src/ui/map/Map.svelte [hunk 2]
```diff
@@ -59,9 +59,11 @@
       }, debounceTimeout);
     };
   }
-
   const debouncedMapZoomBBoxStore = getDebouncedMapZoomBBoxStore();
 
+  /* Custom bounding box */
+  const boundingBox = [-10.5, 47.5, 5.9, 57.8];
```
Comment (vmilitaru) [inline src/ui/map/Map.svelte:65]: Is the custom boundingBox something that should be in the config file ?

Comment (James-Nitron) [reply src/ui/map/Map.svelte:65]: Yeah, I did originally think that but wasn't sure if I should edit it, if you are okay with it I can pop it in there?

**Review by vmilitaru — DISMISSED** [review]
LGTM 👍

File: src/ui/map/Map.svelte [hunk 3]
```diff
@@ -22,6 +22,9 @@
 
   let container;
 
+  /* Custom bounding box */
+  const boundingBox = config.ux.map.customEnglandAndWalesBounds;
```
Comment (patrickjfl) [inline src/ui/map/Map.svelte:26]: same here

**Review by patrickjfl — APPROVED** [review]
LGTM 👍

**Pull request merged**